samples/meetings-token-app/csharp/Service/MeetingService.cs

```diff
@@ -54,10 +55,15 @@ public class MeetingService : IMeetingService
         /// <inheritdoc/>
         public async Task<UserMeetingRoleServiceResponse> GetMeetingRoleAsync(string meetingId, string userId, string tenantId)
         {
-            var serviceUri = this.tenantInfoRepository.GetServiceUrl(tenantId);
-            if (serviceUri == null)
+            var serviceUri = string.Empty;
+
+            if (this.tenantInfoRepository.GetServiceUrl(tenantId) != null)
```

Comment:
Why do we need to call this method twice?
this.tenantInfoRepository.GetServiceUrl(tenantId)

We can assign it's value in a variable and can check for null?